### PR TITLE
ckruse edits from #3275

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds back ckruse edits that were reverted in #3275

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 1]
```diff
@@ -538,10 +539,48 @@ If you can see the following logs from the collector after applying the configur
...
 Make sure you are using collector version `v0.78.0-sumo-0` or higher.
 
+## Collecting logs from other sources
+
+You can find the full list of receivers available on the [Sumo Logic OpenTelemetry Collector][collector_components_docs] page. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.
+
+The Monitoring Job Receiver can be configured to execute a script or command on the collector host. The standard output and error streams from that command will be collected as log record(s).
+
+:::important Prefer purpose-built receivers to Monitoring Jobs
+When available, prefer purpose-built receivers like `sqlquery` to monitoring jobs receiver. This example monitoring job configuration runs SQL queries using the `psql` tool for demonstration purposes as it is well understood.
```
Comment (JV0812) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:549]:
```suggestion
When available, prefer purpose-built receivers like `sqlquery` to monitoring jobs receiver. The below example is about monitoring job configuration that runs SQL queries using the `psql` tool for demonstration purposes as it is well understood.
```

**Review by JV0812 — COMMENTED** [review]
Minor fix

**Review by jpipkin1 — APPROVED** [review]
Approved, contingent on your making @JV0812's suggested update.

**Review by c-kruse — APPROVED** [review]
Thank you for the edit!
+1 for @JV0812's suggestion

**Pull request merged**